src/ChannelServer/Scripting/ScriptManager.cs

```diff
@@ -443,11 +443,13 @@ private int addnpc(IntPtr L)
 			if (name.StartsWith("ETC_") || name.StartsWith("QUEST_"))
 				name = "@dicID_^*$" + name + "$*^";
 
-			var monster = new Monster(monsterId, NpcType.NPC);
```

Comment:
There's nothing wrong with the way you initiate monster here and further down, but you might want to refrain from changing things just so they're to *your* liking. It creates unnecessary diffs, and this being an open-source project not everything will always be like you would normally do it.